include/tscpp/util/TextView.h

```diff
@@ -551,8 +556,11 @@ svto_radix(ts::TextView &src)
   uintmax_t zret{0};
   uintmax_t v;
   while (src.size() && (0 <= (v = ts::svtoi_convert[static_cast<unsigned char>(*src)])) && v < N) {
-    zret *= N;
-    zret += v;
+    auto n = zret * N + v;
+    if (n < zret) { // overflow / wrap
+      return std::numeric_limits<uintmax_t>::max();
```

Comment:
So you are returning maxint if a digit overflows the radix?

In the future there will be places checking for that error value?

Reply:
Yes, it returns the maximum value if there is overflow. This is consistent with the behavior of `strtou` and `strtol`.

For error checking, that's been challenging. I have been through several implementations and I think this is the best option. This is more of an internal method and so depends on the caller to handle the issue. The usual means of detection would be there is data left in `src`, that is not all of the digits were parsed. I'm still finalizing the style to use for `svtoi` and `svtou` which are the "official" interface to numeric parsing. I am currently favoring passing in a `std::error_condition` if overflow checking is desired.